nautobot/ui/src/components/SidebarNav.js

```diff
@@ -0,0 +1,140 @@
+                                            menu[0].startsWith("/") ? (
+                                                <SidebarButton
+                                                    as={ReactRouterLink}
+                                                    key={menu_idx}
+                                                    level={1}
+                                                    to={menu[0]}
+                                                    isLast={
+                                                        menu_idx ===
+                                                        menu_arr.length - 1
+                                                    }
+                                                >
+                                                    {menu[1].name}
+                                                </SidebarButton>
+                                            ) : (
+                                                <Accordion
+                                                    allowMultiple
+                                                    variant="sidebarLevel1"
+                                                    key={menu_idx}
+                                                >
+                                                    <Heading>
+                                                        <AccordionButton
+                                                            isLast={
+                                                                menu_idx ===
+                                                                menu_arr.length -
+                                                                    1
+                                                            }
+                                                        >
+                                                            {menu[0]}
+                                                            <AccordionIcon />
+                                                        </AccordionButton>
+                                                    </Heading>
+                                                    <AccordionPanel>
+                                                        {Object.entries(
+                                                            menu[1].items
+                                                        ).map(
+                                                            (
+                                                                submenu,
+                                                                submenu_idx,
+                                                                submenu_arr
+                                                            ) => (
+                                                                <SidebarButton
+                                                                    as={
+                                                                        ReactRouterLink
```

Comment:
Switching to a 2-space indent and/or splitting some of this out into subcomponents would greatly help readability IMO.